src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitContainer.js

```diff
@@ -30,11 +30,28 @@ const IncidentSplitContainer = ({ FormComponent }) => {
   const dispatch = useDispatch();
   const [parentIncident, setParentIncident] = useState();
   const [directingDepartment, setDirectingDepartment] = useState([]);
+  const [note, setNote] = useState();
   const departments = useSelector(makeSelectDepartments);
   const directingDepartments = useSelector(makeSelectDirectingDepartments);
 
   const [subcategoryGroups, subcategoryOptions] = useSelector(makeSelectSubcategoriesGroupedByCategories);
 
+  const patchData = useMemo(() => {
```

Comment:
I believe my comment on this is lost in `translation`. The suggestion was to use here a useCallback because the patchData is only needed once and only if the splitsing has succeded. Returning a null value when nothing changes would make unnecessary the use of `Object.keys().length` construction.

Reply:
It's here: https://github.com/Amsterdam/signals-frontend/pull/1236#discussion_r519885335